src/unix/tty.c

```diff
@@ -33,12 +34,30 @@ static int orig_termios_fd = -1;
 static struct termios orig_termios;
 static uv_spinlock_t termios_spinlock = UV_SPINLOCK_INITIALIZER;
 
+static int uv__tty_is_slave(const int fd) {
+  int result = 0;
```

Comment:
Can you drop the assignment?  It's somewhat against the house style and what's more, it should help catch bugs when a branch is added below that doesn't assign to `result` (because the compiler - hopefully - complains about it.)